spec/factories/orchestration_stack.rb

```diff
@@ -25,4 +27,18 @@
       x.value = "1"
     end
   end
+
+  factory :orchestration_stack_parameter_openstack_infra_swift_replicas, :parent => :orchestration_stack_parameter_openstack_infra do
+    after :create do |x|
+      x.name = "SwiftReplicas"
+      x.value = "3"
+    end
```

Comment:
as it was pointed to me my @matthewd it's not even nice to define every small nuance in the factory

you can send it in parameters when you create, so having one generic do

FactoryGirl.create(:orchestration_stack_parameter, :name => 'xxx', :value => 'yyy')